cmake/modules/platform/toolcfg/xlc.cmake

```diff
@@ -185,3 +185,19 @@ endif()
 
 set(SPP_CMD ${CMAKE_C_COMPILER})
 set(SPP_FLAGS -E -P)
+
+function(_omr_toolchain_process_exports TARGET_NAME)
```

Comment:
We seem to have this function defined for z/OS slightly above in [1]. Does this new definition make the old one obsolete? If so we should remove the old version.

[1] https://github.com/eclipse/omr/blob/7c8742285db4a1db3b3d55fe6f04cfc80129f20b/cmake/modules/platform/toolcfg/xlc.cmake#L172-L183

Reply:
Yeah, I just noticed that as well. I'm looking now to see if the export format is the same.

Reply:
Any update @dnakamura ?